tools/repl/swift/CMakeLists.txt

```diff
@@ -10,8 +10,13 @@ if( CMAKE_SYSTEM_NAME MATCHES "Linux" )
   set(CMAKE_INSTALL_RPATH "$ORIGIN/../lib/swift/linux:${CMAKE_INSTALL_RPATH}")
 endif()
 
-# Override locally, so the repl is ad-hoc signed.
-set(LLVM_CODESIGNING_IDENTITY "-")
+# LLVM_CODESIGNING_IDENTITY being set makes cmake attempt to invoke xcrun when
+# crosscopmiling to Darwin. So if we can't find xcrun then opt out of attempting
+# to do so.
+if (CMAKE_XCRUN)
```

Comment:
I don't really like that we depend on an implementation detail of code signing (that it'll go through xcrun) to disable this. Is there something better we can check? Or even better, can we fix `llvm_codesign` in LLVM to take cross compiling into account?

Reply:
`CMAKE_HOST_APPLE AND APPLE`? I can't say I know how the Apple code signing works to solve this, just that it's not required elsewhere.

Reply:
What about using `CMAKE_HOST_APPLE` in https://github.com/llvm/llvm-project/blob/master/llvm/cmake/modules/AddLLVM.cmake#L1685 ?

Reply:
> What about using CMAKE_HOST_APPLE
+1 but upstream maybe better fixed after the 9.0 branch cut